### DISCUSSION TITLE
Welcome to pair_allegro Discussions!

### DISCUSSION THREAD

**Discussion by Linux-cpp-lisp** [mir-group/pair_nequip_allegro#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Hongyu-yu** [comment]
Hello everyone! I'm [Hongyu Yu](https://www.researchgate.net/profile/Hongyu-Yu-6) from Fudan University under supervision of [Hongjun Xiang](https://www.researchgate.net/profile/H-Xiang) and excited to connect with more people in this community. 

My works focus on the application of ML technique in materials especially magnetic material.

Allegro is one of the best ML potentials nowadays and [my recent work](http://arxiv.org/abs/2211.11403) extents allegro into spin-allegro with [Time-reversal e3nn](https://github.com/Hongyu-yu/T-e3nn) to consider the spin degrees in magnetic material.

As for NPT parallel simulations, I submit related PRs to make Allegro complete. Hope it helps!

Open-minded to any kind of connections and my email is hongyuyu20@fudan.edu.cn :)

With some experience of using allegro and developing on it, I am willing to contribute to the building of this community.
